database/migrations/2014_12_23_130853_version_and_revision_talks.php

```diff
@@ -3,8 +3,8 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class VersionAndRevisionTalks extends Migration {
-
+class VersionAndRevisionTalks extends Migration
```

Comment:
The problem is the JSON is invalid, there's a trailing comma after the last entry. I need to add something to Nitpick to get it to post a comment if it fails to parse a JSON config.